packages/web/components/PageHeader.tsx

```diff
@@ -71,13 +70,20 @@ const SubMenuItem: React.FC<React.ComponentProps<typeof MetaLink>> = ({
         alignItems="center"
         width="8.25rem"
         height="8.25rem"
+        backgroundColor="rgba(255,255,255,0.08)"
+        borderRadius="5px"
         textDecoration="none"
         variant="link"
-        fontFamily="mono"
+        fontFamily="body"
+        fontWeight="normal"
         color="whiteAlpha.700"
         margin={3}
         className="filter-effect"
-        backgroundImage={`url(${MetaBoxButton})`}
+        // backgroundImage={`url(${MetaBoxButton})`}
```

Comment:
Good work! However, we don't need this `//`

Reply:
Yeah, I intended to a broader refactor combining these two components, but ran out of time, so I just called it good here. It is a dramatic visual improvement I'd say